### PR TITLE
Fix #81513 m.dcinside.com

### LINKED ISSUE #81513: mupload.dcinside.com
### Issue URL (Incorrect Blocking)
 [https://mupload.dcinside.com/write_new.php](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fmupload.dcinside.com%2Fwrite_new.php)
### Comment
Username: @sharbet
> Problem explained here: 
> https://forum.adguard.com/index.php?threads/bug-adguard-tracking-filter-in-android-prevents-posting-for-dcinside-com.42769/
> 
> When I try to post on that website, I cannot post on that website, because Adguard Tracking filter won't allow me to.
> Disabling Tracking filter allows me to post without problems.

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>
<details>
  <summary>Screenshot 2</summary>

  Screenshot 2
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard for Windows v7.6 nightly 40
Browser: | Firefox
AdGuard driver: | WFP
Stealth mode options: | Hide your search queries, <br>Send Do-Not-Track header, <br>Remove X-Client-Data header from HTTP requests, <br>Strip URLs from tracking parameters, <br>Block third-party Authorization header, <br>Self-destructing first-party cookies (4320), <br>Self-destructing third-party cookies (1), <br>Disable cache for third-party requests, <br>Block Push API, <br>Block Location API, <br>Block Flash, <br>Block Java, <br>Hide your Referrer from third-parties (mupload.dcinside.com), <br>Hide your User-Agent ()
DNS filtering: | server: `https://security.cloudflare-dns.com/dns-query`<br>filters: `User rules, https://abp.oisd.nl/`
Filters: | <b>Ad Blocking:</b><br/> AdGuard Mobile Ads, <br/>AdGuard Base<br/><br/><b>Privacy:</b><br/> AdGuard Tracking Protection<br/><br/><b>Social Widgets:</b><br/> AdGuard Social Media<br/><br/><b>Annoyances:</b><br/> AdGuard Annoyances<br/><br/><b>Other:</b><br/> AdGuard Experimental<br/><br/><b>Language-specific:</b><br/> List-KR, <br/>YousList, <br/>AdGuard Japanese
Userscripts: | https://userscripts.adtidy.org/beta/popup-blocker/2.5/popupblocker.user.js,<br>https://userscripts.adtidy.org/beta/adguard-extra/1.0/adguard-extra.user.js

### PR DESCRIPTION
**Pull request by green1052**
# Creating the pull request

> Please include a summary of the change and which issue is fixed\
> If the related issue does not exist in our repository, please create it before making pull request\
> It is highly recommended to use our [Web Reporting Tool](https://kb.adguard.com/en/technical-support/reporting-tool) instead of creating an issue on GitHub directly\
> Please note, that we verify every pull request manually, so it may take time to apply it

### To avoid invalid pull requests, please check and confirm following terms

- [X] This is not an ad/bug report;
- [X] My code follows the [guidelines](https://github.com/AdguardTeam/AdguardFilters/blob/master/CONTRIBUTING.md) and [syntax](https://kb.adguard.com/general/how-to-create-your-own-ad-filters) of this project;
- [X] I have performed a self-review of my own changes;
- [X] My changes do not break web sites, apps and files structure.

### If the problem does not fall under any category that is listed here, please write a comment below in corresponding section

- [ ] Missed ads or ad leftovers;
- [ ] Website or app doesn't work properly;
- [ ] AdGuard gets detected on a website;
- [ ] Missed analytics or tracker;
- [ ] Social media buttons — share, like, tweet, etc;
- [ ] Annoyances — pop-ups, cookie warnings, etc;
- [X] Filters maintenance.

### Enter the issue address

https://github.com/AdguardTeam/AdguardFilters/issues/81513

#### If possible, a screenshot of a page or application should not be cropped too much. Otherwise, it is not always clear where the element is located

0. **DO NOT** upload screenshots with sexually explicit material on GitHub directly.\
 Instead, upload it to a third-party image hosting and post the link to it without preview (!) here.\
 Also, mention if the link leads to NSFW content;

1. Add screenshots of the problem. You can drag and drop images or paste them from clipboard

    Use `<details> </details>` tag to hide screenshots under the spoiler;

2. Describe the issue in detail until it is absolutely clear from the screenshot what the problem is

    You can also indicate any other information that you think the developers should know.

**Warning:** Please remove personal information before uploading screenshots!

1. Your comment

2. Screenshots

<details>

<summary>Screenshot 1:</summary>

</details>

<details>

<summary>Screenshot N:</summary>

</details>

### Terms

- [X] By submitting this issue, I agree that pull request does not contain private info and all conditions are met

### CODE REVIEW

File: SpywareFilter/sections/mobile_allowlist.txt [hunk 1]
```diff
@@ -15,10 +15,6 @@
 @@||google-analytics.com^$app=jp.co.alc.eow
 @@||tr.line.me^$app=jp.co.alc.eow
 @@||tr.outbrain.com^$app=jp.co.alc.eow
-! https://github.com/AdguardTeam/AdguardFilters/issues/81513
-@@||google-analytics.com/analytics.js$script,domain=m.dcinside.com|mupload.dcinside.com
-@@||ssl.google-analytics.com/ga.js$script,domain=m.dcinside.com|mupload.dcinside.com
-@@||dcinside.com$cookie=_ga
```
Comment (krystian3w) [inline SpywareFilter/sections/mobile_allowlist.txt:21]: Remove last line maybe enforce bake cookie on every refresh instead one of year (if `_ga` is still removed globally).

Comment (green1052) [reply SpywareFilter/sections/mobile_allowlist.txt:21]: fixed

**Comment by Alex-302** [comment]
What exactly this PR fixes? `Trusted` scriptlets are not supported by release versions of all AdGuards.

**Comment by green1052** [comment]
@piquark6046 can you help me?

**Comment by krystian3w** [comment]
Works in beta addon: https://github.com/AdguardTeam/AdguardBrowserExtension/releases/tag/v4.1.52-beta

No tested at all in "hotfix": https://github.com/AdguardTeam/AdguardBrowserExtension/releases/tag/v4.1.52 (~14 december 2022).

**Comment by piquark6046** [comment]
If any string can resolve this issue as you said in the email conversation with @green1052, you can use `set-cookie`.

File: SpywareFilter/sections/mobile_allowlist.txt [hunk 2]
```diff
@@ -16,8 +16,6 @@
 @@||tr.line.me^$app=jp.co.alc.eow
 @@||tr.outbrain.com^$app=jp.co.alc.eow
 ! https://github.com/AdguardTeam/AdguardFilters/issues/81513
-@@||google-analytics.com/analytics.js$script,domain=m.dcinside.com|mupload.dcinside.com
-@@||ssl.google-analytics.com/ga.js$script,domain=m.dcinside.com|mupload.dcinside.com
```
Comment (Alex-302) [inline SpywareFilter/sections/mobile_allowlist.txt:20]: Better wrap by the directive. Exclusions still required for `content blocker for Android`(not the AdGaurd for Android app).
```
!#if (adguard_ext_android_cb)
@@||google-analytics.com/analytics.js$script,domain=m.dcinside.com|mupload.dcinside.com
@@||ssl.google-analytics.com/ga.js$script,domain=m.dcinside.com|mupload.dcinside.com
!#endif
```

**Review by Alex-302 — CHANGES_REQUESTED** [review]

File: SpywareFilter/sections/general_extensions.txt [hunk 3]
```diff
@@ -11,6 +11,8 @@
 ! https://segment.com/docs/connections/sources/catalog/libraries/website/javascript/identity/
 ! TODO: replace by key removing scriptlet when it will be implemented
 reuters.com#%#//scriptlet('set-local-storage-item', 'ajs_anonymous_id', 'null')
+! https://github.com/AdguardTeam/AdguardFilters/issues/81513
+m.dcinside.com,upload.dcinside.com#%#//scriptlet('trusted-set-cookie', '_ga', 'a', '1year')
```
[the remark this answers is not in the document]
Comment (piquark6046) [reply SpywareFilter/sections/general_extensions.txt:15]: I did not test. But does `m.dcinside.com,upload.dcinside.com#%#//scriptlet('set-cookie', '_ga', 'true')` not work?

**Comment by green1052** [comment]
ok

**Review by Alex-302 — APPROVED** [review]

**Pull request merged**